docs-web/configuration/transportserver-resource.md

```diff
@@ -181,6 +181,64 @@ port: 8443
      - Yes
 ```
 
+
+### Upstream.Healthcheck
```

Comment:
could you also add the field `healthCheck` to the table for https://deploy-preview-1384--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/configuration/transportserver-resource/#upstream which links to this section ?

Reply:
could you update the upstream fields table with the health check field? 

this is what we have for the VS:
![image](https://user-images.githubusercontent.com/2199726/108548442-b0b64080-72a0-11eb-8f34-0143f6816eb8.png)